lib/plugins/features/movies/controller.js

```diff
@@ -1,9 +1,56 @@
 'use strict';
 
-const Movie = require('../../../models/movie');
+// Validators
+const Movie     = require('../../../models/movie');
+const MovieList = require('../../../validators/movielist');
+
+// Libraries for SQL requests
+const Boom = require('@hapi/boom');
+const Knex = require('../../../libraries/knex');
 
 exports.create = async (payload) => {
   const movie = await new Movie().save(payload);
 
   return new Movie({ id: movie.id }).fetch();
 };
+
+exports.retrieve = async (query) => {
+  // Perform validation inside method since Hapi doesn't support
+  // validation on GET requests
```

Comment:
actually, hapi does allow you to do validation on the query params for GET requests. for instance, we do this for many of our endpoints in lob-api such as our [postcard endpoints](https://github.com/lob/lob-api/blob/4a0fa057748c976166c443dfecf4689da3aa0aa0/src/plugins/features/postcards/index.js#L261)

Reply:
if possible, i would try to move the validator logic away from the controller if hapi can actually validate query params. if that takes too much time for you to implement, then we can keep this approach for now

Reply:
Just made this fix, I think my confusion from before was that I was trying to validate the `payload` parameter, and I read online that no validation can occur for the `payload` parameter on `GET` requests, but I see now that it's actually called query